### DISCUSSION TITLE
AC Light

### DISCUSSION THREAD

**Discussion by arthurlorga** [veista/smartthings#109] [Q&A]
I am using the Samsung Windfree Connect AC Unit (model AR12CVFAMWKNAZ) and I am not able to control the display light on it from home assistant. Is there any additional confuguration I need to run/perform so the option to turn off the unit display will show up? The custom integration states that AC Unit light should be an option. Any help is greatly appreciated. Thanks.

**Comment by bortuzar** [comment]
I am also looking for this. Did you find it?

**Comment by uncredited** [comment]
Stumbled on this thread while searching for something else. 
As this discussion is very old I assume you've already found a solution. 
If not I hope this might help you a little. 
The light control for my Quiet Comfort 35 stopped working with the SmartThings integration and what I found was that sending "Light_Off" turned the light ON and sending "Light_On" turned it off. ie the reverse of what you'd think.
I ended up vibe-coding a custom integration to handle the light on my unit. 
I hope this helps, if you haven't found another way to solve your issues. :)
I apologize for reviving such an old discussion.
